packages/stopwatch/package.json

```
@@ -0,0 +1,51 @@
+{
+	"name": "@sapphire/stopwatch",
+	"version": "0.0.0",
+	"description": "Accurately measure passing time.",
+	"author": "@sapphire",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js"
+	},
+	"homepage": "https://github.com/sapphire-project/utilities/tree/main/packages/stopwatch",
+	"scripts": {
+		"test": "jest",
+		"lint": "eslint src tests --ext ts --fix -c ../../.eslintrc",
+		"build": "tsc -p src",
```

```suggestion
		"build": "tsc -b src",
```

-b is build targets
-p is project

While both do work in this case, -b is what we use elsewhere.